From 042934e78676a7854038be560e8f8a7d05552b84 Mon Sep 17 00:00:00 2001
From: Justin Ruggles <justin.ruggles@gmail.com>
Date: Wed, 10 Aug 2011 14:07:35 -0400
Subject: [PATCH 28/70] Remove incorrect info in documentation of AVCodecContext.bits_per_raw_sample.

bits_per_raw_sample is used in video as well, where sample_fmt is not used.
(cherry picked from commit d271d5b2152cafe540f3ab71d3be6ce8636d2fd6)

Signed-off-by: Anton Khirnov <anton@khirnov.net>
---
 libavcodec/avcodec.h |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/libavcodec/avcodec.h b/libavcodec/avcodec.h
index 9a3076a..0269892 100644
--- a/libavcodec/avcodec.h
+++ b/libavcodec/avcodec.h
@@ -2559,7 +2559,6 @@ typedef struct AVCodecContext {
 
     /**
      * Bits per sample/pixel of internal libavcodec pixel/sample format.
-     * This field is applicable only when sample_fmt is AV_SAMPLE_FMT_S32.
      * - encoding: set by user.
      * - decoding: set by libavcodec.
      */
-- 
1.7.4.1

